ipmi/stress/driver/metadata

```diff
@@ -0,0 +1,12 @@
+[General]
+name=ipmi/stress/driver
+owner=Rachel Sibley <rasibley@redhat.com>
+description=IPMI driver load stress test
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+dependencies=dmidecode
```

Comment:
Please move dmidecode as a soft dependency, it's not supported on ppc64le, I have this built into the test, you will see it's aborting on this arch.

Reply:
move "dmidecode" ipmi metadata tests to softDependencies